packages/ui/src/message-room-details-dialog/components/ArchiveChannel.jsx

```diff
@@ -1,7 +1,37 @@
 import ChannelModal from "./ChannelModal";
 import styles from "./archive-channel.module.css";
-
+import { useEffect, useState } from "react";
+import axios from "axios";
 const ArchiveChannel = ({ closeEdit }) => {
+  const room = window.location.href.split("/").at(6);
+  const organizationID = localStorage.getItem("currentWorkspace") || null;
+  const BASE_URL = "https://chat.zuri.chat";
+  const user = JSON.parse(sessionStorage.getItem("user")) || null;
+
+  const [data, setData] = useState(null);
+  useEffect(() => {
+    axios
+      .get(`${BASE_URL}/api/v1/org/${organizationID}/rooms/${room}`)
+      .then(res => {
+        setData(res.data.data);
+      })
+      .catch(e => console.log(e));
+  });
+  const handleArchiveChannel = () => {
+    if (data !== null) {
+      const newData = { ...data, is_archived: true };
+      axios
+        .put(
+          `${BASE_URL}/api/v1/org/${organizationID}/members/${user.id}/rooms/${room}`,
+          newData
+        )
+        .then(res => {
+          console.log(res);
```

Comment:
don't log the response
send it as a toast notification to the user

Reply:
Can i install the react-toastify library ?

Reply:
there's a toast library already installed

Reply:
I can't see it , what's the name of the package ?